web/packages/teleport/src/Locks/CreateLock.tsx

```diff
@@ -0,0 +1,171 @@
+      const message = messageRef?.current?.value;
+      const ttl = ttlRef?.current?.value;
+      if (message) lockData.message = message;
+      if (ttl) lockData.ttl = ttl;
+      await createLock(clusterId, lockData)
+        .then(() => {
+          setTimeout(() => {
+            // It takes longer for the cache to be updated when adding locks so
+            // this waits 1s before redirecting to fetch the list again.
+            history.push(cfg.getLocksRoute(clusterId));
+          }, 1000);
+        })
+        .catch((e: ApiError) => {
+          setError(e.message);
+        });
+    });
+  }
+
+  function onRemove(name) {
+    const index = selectedLockTargets.findIndex(target => target.name === name);
+    selectedLockTargets.splice(index, 1);
+    setSelectedLockTargets([...selectedLockTargets]);
+  }
+
+  return (
+    <SlidePanel
+      position={panelPosition}
+      closePanel={() => setPanelPosition('closed')}
+    >
+      <div>
+        {error && <Alert kind="danger" children={error} data-testid="alert" />}
+        <Flex alignItems="center">
+          <ArrowBack
+            fontSize={25}
+            mr={3}
+            onClick={() => setPanelPosition('closed')}
+            style={{ cursor: 'pointer' }}
+          />
+          <Box>
+            <Text typography="h4" color="light" bold>
+              Create New Lock
+            </Text>
+          </Box>
+        </Flex>
+
+        <StyledTable
+          data={selectedLockTargets}
+          css={`
+            margin-top: 3rem;
+          `}
+          columns={[
+            {
+              key: 'type',
+              headerText: 'Type',
+              isSortable: false,
+            },
+            {
+              key: 'name',
+              headerText: 'Name',
+              isSortable: false,
+            },
+            {
+              altKey: 'remove-btn',
+              render: ({ name }) => (
+                <Cell align="right">
+                  <Trash
+                    fontSize={13}
+                    borderRadius={2}
+                    p={2}
+                    onClick={onRemove.bind(null, name)}
+                    css={`
+                      cursor: pointer;
+                      background-color: #2e3860;
+                      border-radius: 2px;
+                      :hover {
+                        background-color: #414b70;
+                      }
+                    `}
+                    data-testid="trash-btn"
+                  />
+                </Cell>
+              ),
+            },
+          ]}
+          emptyText="No Targets Found"
+        />
+        <Box mt={3}>
+          <Text mr={2}>Message: </Text>
+          <Input
+            placeholder={`Going down for maintenance`}
+            ref={messageRef}
+            data-testid="description"
+          />
+        </Box>
+        <Box mt={3}>
+          <Text mr={2}>TTL: </Text>
+          <Input placeholder={`5h`} ref={ttlRef} data-testid="ttl" />
```

Comment:
I actually ran into this myself because I didn't know all of the suffixes for a TTL. In a follow-up there will be an expansion on this UI where you can input an `expires` date instead of a `ttl`. But I can add some more detail to the placeholder to illustrate all allowed suffixes.

Reply:
I think just two examples + "empty = never" would be enough to get the context of how it works. Maybe its already enough given our TTL syntax elsewhere. Just a thought

Reply:
Added!